modules/client/CHANGELOG.md

```diff
@@ -13,6 +13,19 @@ TEMPLATE:
 
 -->
 ## [UPCOMING]
+## [1.4.3]
```

Comment:
We are adding a new feature, this should be a minor version bump, not just a patch

Technically, there are breaking changes and this would have to be a major release, but the App team is not using it and we don't want to break the relationship with the OSx version 1